packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx

```diff
@@ -28,9 +28,21 @@ export interface GuideCardConstants {
   telemetryId: string;
   order: number;
   icon: string;
+  // the guide will open a specific modal ESApiModal
+  openEndpointModal?: boolean;
 }
 
 export const guideCards: GuideCardConstants[] = [
+  {
```

Comment:
I thought this was supposed to be shown for all use cases, not just search. Can you double check?

Reply:
@kevinsweet can you confirm for me that the new card should only be in the search use case for now? Thanks!